README.md

```diff
@@ -2,13 +2,24 @@
 [![](https://vsmarketplacebadge.apphb.com/version/xcomponent.xcomponent.svg)](https://marketplace.visualstudio.com/items?itemName=XComponent.xcomponent)
 [![Build Status](https://travis-ci.org/xcomponent/vscode-xcomponent.svg?branch=master)](https://travis-ci.org/xcomponent/vscode-xcomponent)
 
-## Visual Studio Code extension for XComponent
-This repository contains the code of:
-* the vscode extension
-* the component viewer used in the extension
+# Visual Studio Code extension for [XComponent](http://www.xcomponent.com/)
+
+This repository contains:
+* [xcomponent extension](https://github.com/xcomponent/vscode-xcomponent/blob/master/extension/README.md)
```

Comment:
I change the introduction.
As we talked about yesterday, I have the same introduction on the main readme et the extension readme (used by vscode store).